python/ingestion/gcs_to_bq_util.py

```diff
@@ -7,20 +7,23 @@
 from google.cloud import bigquery, storage
 
 
-def append_dataframe_to_bq(
-        frame, dataset, table_name, column_types=None, col_modes=None, project=None):
-    """Appends the provided DataFrame to the table specified by
-       `dataset.table_name`. Automatically adds an ingestion time column.
+def __convert_frame_to_json(frame, column_types):
```

Comment:
Sorry to get picky, but could you add a docstring that indicates this function modifies column_types?

Reply:
No problem. It was a bit odd for this function to add the ingestion timestamp. Moved this logic into a separate function and placed it where the method description gives details on what's going on here.